packages/alfa-aria/src/get-role.ts

```diff
@@ -18,7 +18,10 @@ for (const role of values(Roles)) {
 
 const featuresByElement = new Map<string, Feature>();
 
-for (const feature of values(Features)) {
+for (const feature of values(Features.HTML)) {
+  featuresByElement.set(feature.element, feature);
+}
+for (const feature of values(Features.SVG)) {
```

Comment:
Features will need to be distinguished on namespace as SVG features that overlap with HTML features will otherwise overwrite these.